code/datums/gamemode/misc_gamemode_procs.dm

```diff
@@ -263,16 +263,20 @@ proc/name_wizard(mob/living/carbon/human/wizard_mob)
 	if(H)
 		qdel(H)
 
-/proc/add_law_zero(mob/living/silicon/ai/killer)
+/proc/add_law_zero(mob/living/silicon/killer)
 	var/law = "Accomplish your objectives at all costs."
-	var/law_borg = "Accomplish your AI's objectives at all costs."
+	if(isAI(killer))
```

Comment:
I think you got that one backwards ; it would give AIs `"Accomplish your AI's objectives at all costs."` and borgs `"Accomplish your objectives at all costs."`

Reply:
The first arg is the law to give the mob, and the second arg is the law to give connected robots, so this is the correct order.